### PR TITLE
Android Bug Reporting Copy

### PR DESCRIPTION
**Pull request by skylarbarrera**
Changed the copy on android for the beta, added OS & app version to feedback email subject line

@christianbaroni looks the same as before on iOS. I was initially gonna add another list item but I think this makes more sense.

I prefer with the extra comma but it's a personal preference, lmk you thoughts

no comma: https://cloud.skylarbarrera.com/Screen-Shot-2020-11-20-12-34-12.16.png

with comma: https://cloud.skylarbarrera.com/Screen-Shot-2020-11-20-12-34-50.55.png

### CODE REVIEW

**Comment by brunobar79** [comment]
@skylarbarrera  can u add a screenshot of how it looks here so @christianbaroni  can peep ?

File: src/components/settings-menu/SettingsSection.js [hunk 1]
```diff
@@ -224,8 +224,8 @@ export default function SettingsSection({
           value={SettingsExternalURLs.twitter}
         />
         <ListItem
-          icon={<Emoji name="speech_balloon" />}
-          label="Feedback and Support"
+          icon={<Emoji name={ios ? 'speech_balloon' : 'bug'} />}
+          label={ios ? 'Feedback and Support' : 'Feedback, Support, & Bugs'}
```
Comment (brunobar79) [inline src/components/settings-menu/SettingsSection.js:228]: There's an extra `,` on the android copy

**Review by brunobar79 — CHANGES_REQUESTED** [review]

File: src/components/settings-menu/SettingsSection.js (hunk 1 above)
Comment (skylarbarrera) [reply src/components/settings-menu/SettingsSection.js:228]: force of habit 🙆‍♂️

**Review by brunobar79 — APPROVED** [review]
100

**Pull request merged**